City-of-Helsinki/drupal-module-helfi-api-base
# UHF-5041: Stdout logger

To test this:

- Pull the latest docker image: `docker compose pull`, then `make stop && make up`
- `composer require drupal/helfi_api_base:dev-UHF-5041` or if that doesn't work: `composer reinstall --prefer-source drupal/helfi_api_base` and then checkout the `UHF-5041` branch in `public/modules/contrib/helfi_api_base`.
- You might have to install `drupal/filelog` module manually if it wasn't installed with composer require earlier (`composer require drupal/filelog`).
- `drush updb && drush cr`

You can test this by adding `\Drupal::logger('helfi_api_base')->error('Something');` to `src/Controller/DebugController.php` file and then opening the `/admin/debug` page. 

Verify that the error message is sent to container logs: `docker compose logs app`. For example:
```
helfi-kymp-app      | [31. maaliskuuta 2022] [Error] [helfi_api_base] [client: 192.168.64.11, helfi-admin] Test
```
